General Review

uguryuruk opened the pull request:
I have made the same changes in my previous commits. I am reviewing the whole snippets comparing with original docs.

anburocky3 commented on src/templates/bs5/components/alert/default.html:
```diff
@@ -1,3 +1,3 @@
-<div class="alert alert-${1:primary|secondary|success|danger|warning|info|light|dark}" role="alert">
+<div class="alert ${1|alert-primary,alert-secondary,alert-success,alert-danger,alert-warning,alert-info,alert-light,alert-dark|}" role="alert">
```
Do we need really this?

Can we keep this:
```html
<div class="alert alert-${1:primary|secondary|success|danger|warning|info|light|dark}" role="alert">
```

Instead of this:
```html
<div class="alert ${1|alert-primary,alert-secondary,alert-success,alert-danger,alert-warning,alert-info,alert-light,alert-dark|}" role="alert">
```

This will trigger like this: 

```html
<div class="alert alert-CURSOR-HERE" role="alert"></div>
```

why do we need to type `alert-` unnecessarily?

uguryuruk replied:
Yes we need to Do that otherwise suggestions come empty. I guess after - our suggestions don't work. You can try on vs code.

anburocky3 replied:
Yes, with your method, the suggestions are displayed. Thank you.

anburocky3 commented on src/templates/bs5/components/form/checkbox.html:
```diff
@@ -1,6 +1,12 @@
+</div>
+<div class="form-check">
+  <input class="form-check-input" type="checkbox" value="$5" id="$4" ${6:checked}>
+  <label class="form-check-label" for="$3">
```
`for="$4"`  instead of `for="$3"`

anburocky3 reviewed (changes requested):
Kindly respond to the reviews and once fixing this, we can merge this. 

Thank you for your contribution

uguryuruk commented:
@anburocky3  I have fixed all issues in the request. I think the order still works even if there are some gaps (for example it jumps from 4 to 6 if there is no 5).

In cards, I have made the alt attribute and the title same, in order to fill alt quickly.

Except one, col-$sm-$4 style is not working. So I thought that at least I can give the screen size options for columns, then user can change the number. So I have added new end points just in the end of col-sm-4$. If you have another suggestion, I may do it.
